app/components/works/contributor_role_component.rb

```diff
@@ -3,18 +3,20 @@
 module Works
   # Renders a widget for selecting a contributor role
   class ContributorRoleComponent < ApplicationComponent
-    def initialize(form:, data_options:)
+    def initialize(form:, data_options:, contributor_type:, visible:)
       @form = form
+      @contributor_type = contributor_type # person or organization
+      @visible = visible # if drop down starts out as hidden/disabled
       @data_options = data_options
```

Comment:
We now render two drop down menus for the contribute role type, one for "person" and one for "organization".  Only one starts out as visible depending on the current state of the entered data.